simpleclient_pushgateway/src/main/java/io/prometheus/client/exporter/PushGateway.java

```diff
@@ -110,20 +110,19 @@
    * @param authority host:port or ip:port of the Pushgateway.
    */
   public PushGateway(String authority) {
```

Comment:
This should be "host" or similar, as most users will never have heard of "authority"